code/ATMOSPHERICS/hvac/cryotheum_resonator.dm

```diff
@@ -161,7 +163,11 @@
 			update_icon()
 
 /obj/machinery/cryotheum_resonator/attack_hand(mob/user as mob)
-	src.add_fingerprint(user)
+	if(issilicon(user))
+		add_hiddenprint(user)
```

Comment:
Not sure if this is actually necessary - line 59 in cyborg.dm already adds a hiddenprint so you could likely just return here (although I don't even think silicons can add fingerprints but I'm sure there was an issue being corrected here)